lib/utils/config.ts

```diff
@@ -14,11 +14,11 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 import Promise = require('bluebird');
-import ResinSdk = require('resin-sdk');
+import ResinSdk = require('resin-sdk-preconfigured');
 import _ = require('lodash');
 import deviceConfig = require('resin-device-config');
 
-const resin = ResinSdk.fromSharedOptions();
+const resin = ResinSdk;
```

Comment:
This is redundant -- why not remove this line as you did on the file above? Same for the other files.

Reply:
I reverted this file back to using resin-sdk